.dockerignore

```diff
@@ -1,6 +1,13 @@
-/target
-/book
-/assets
-/.github
-/data
-*.tar.gz
\ No newline at end of file
+# ignore all
+*
+
+# exclude source files
+!/bin
+!/crates
+!/testing
+!book.toml
+!Cargo.lock
+!Cargo.toml
+!Cross.toml
+!deny.toml
+!Makefile
```

Comment:
Please include the licenses too

Reply:
FYI right now (before my PR) the licenses are copied during the reth binary build stage of the Dockerfile, but are not copied into the final runner container. Which means that the end user that pulls the reth image won't find the licenses in it. It seems pointless to me to copy the licenses during build time only, would you like that I add them to the final runner container by copying them across the various build stages?